edge-modules/azure-monitor/src/README.md

```diff
@@ -0,0 +1,152 @@
+    "Name": "docker_container_disk_write_bytes",
+    "Value": 0.0,
+    "Label": {
+        "name": "AzureMonitorForIotEdgeModule"
+    }
+}]
+```
+
+
+## Allow and Disallow Lists:
+
+`AllowedMetrics` and `BlockedMetrics` are space or comma separated lists of metric selectors. A metric will match the list and be included or excluded if it matches one or more metrics in either list. 
+
+Metric selectors use a format similar to a subset of the PromQl query language. They consist of three parts:
+
+``` metricToSelect{quantile=0.5,otherLabel=~Re[ge]*|x}[http://VeryNoisyModule:9001/metrics] ```
+
+1. Metric name (`metricToSelect`). This component is required.
+    - Wildcards `*` (any characters) and `?` (any single character) can be used in metric names. For example, `*CPU` would match `maxCPU` and `minCPU` but not `CPUMaximum`. `???CPU` would match `maxCPU` and `minCPU` but not `maximumCPU`.
+    - This component is required.
+2. Label based selectors (`{quantile=0.5,otherLabel=~Re[ge]*|x}`). 
+    - Multiple metric=value can be included in the curly brackets, they should be comma separated.
+    - A metric will be matched if at least all labels in the selector are present and also match.
+    - Like PromQl, the following matching operators are allowed.
+        - `=` Match labels exactly equal to the provided string (case sensitive).
+        - `!=` Match labes not exactly equal to the provided string.
+        - `=~` Match labels to a provided regex. ex: `label~=CPU|Mem|[0-9]*`
+        - `!=` Match labels which do not fit a provided regex.
+    - (Regex note: A [.NET regex engine](https://docs.microsoft.com/en-us/dotnet/standard/base-types/regular-expression-language-quick-reference) is used. PromQL uses RE2, be ware)
+    - Regexs are fully anchroed (A ^ and $ are automatically added to the start and end of each regex)
+    - This component is optional.
+3. Endpoint selector (`[http://VeryNoisyModule:9001/metrics]`).
+    - The URL should exactly match a URL listed in `MetricsEndpointsCSV`.
+    - This component is optional.
+
+A metric must match all parts of a given selector to be selected. I.e. a metric must match the name **and** have all the same tags with matching values **and** come from the given endpoint. `mem{quantile=0.5,otherLabel=foobar}[http://VeryNoisyModule:9001/metrics]` would not match the selector `mem{quantile=0.5,otherLabel~=foo|bar}[http://VeryNoisyModule:9001/metrics]`. Multiple selectors should be used to create or-like behavior instead of and-like behavior.
+
+
+For example, to allow the metric `mem` from the host `host1` regardless of tags but only allow the same metric from `host2` with the tag `agg=p99`, the following selector could added to `AllowedMetrics`:
+
+```
+mem{}[http://host1:9001/metrics] mem{agg="p99"}[http://host2:9001/metrics]
+```
+
+Or, to allow the metrics `mem` and `cpu` regardless of tags or endpoint, the following could be added to `AllowedMetrics`:
+```
+mem cpu
+```
+
+
+## Contributing
```

Comment:
We can remove the ##Contributing, ##Building Code, and ## Tests sections.

Reply:
Removed.